Makefile

```
@@ -23,12 +23,16 @@
 	@echo "🧹  Diagramas removidos"
 
 
-.PHONY: pwa-deps api-deps
+.PHONY: pwa-deps api-deps dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)